frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -73,6 +74,26 @@ const WARNING_TYPE_RENDERER = {
             </>
         )
     },
+    event_timestamp_in_future: function Render(warning: IngestionWarning): JSX.Element {
+        const details = warning.details as {
+            timestamp: string
+            sentAt: string
+            offset: string
+            now: string
+            result: string
+        }
+        return (
+            <>
+                Timestamp computed to <code>{details.result}</code> from the following input:
```

Comment:
Is there anywhere we can highlight how and why result is calculated the way that it is? Just a comment around time skew etc. I found it very confusing when I was looking at this first.

Reply:
Definitely. We have an autogenerated `docs` links to `https://posthog.com/manual/data-management#an-event-was-sent-more-than-23-hours-in-the-future`, we need to contribute that documentation change. That's where we'll explain this.

Reply:
👍